graphql/src/main/java/com/linecorp/armeria/server/graphql/GraphQLServiceBuilder.java

```diff
@@ -0,0 +1,260 @@
+        return configureGraphQL(ImmutableList.copyOf(requireNonNull(configurers, "configurers")));
+    }
+
+    /**
+     * Adds the {@link GraphQLConfigurator} consumers.
+     */
+    public GraphQLServiceBuilder configureGraphQL(
+            Iterable<? extends GraphQLConfigurator> configurers) {
+        graphQLBuilderConsumers.addAll(requireNonNull(configurers, "configurers"));
+        return this;
+    }
+
+    /**
+     * Sets whether the service executes service methods using the blocking executor.
+     */
+    public GraphQLServiceBuilder useBlockingTaskExecutor(boolean useBlockingTaskExecutor) {
+        this.useBlockingTaskExecutor = useBlockingTaskExecutor;
+        return this;
+    }
+
+    /**
+     * Creates a {@link GraphQLService}.
+     */
+    public GraphQLService build() {
+        final TypeDefinitionRegistry registry = typeDefinitionRegistry(schemaFiles.build());
+
+        final RuntimeWiring runtimeWiring = buildRuntimeWiring(runtimeWiringConfigurators.build());
+        GraphQLSchema schema = new SchemaGenerator().makeExecutableSchema(registry, runtimeWiring);
+        final List<GraphQLTypeVisitor> typeVisitors = this.typeVisitors.build();
+        for (GraphQLTypeVisitor typeVisitor : typeVisitors) {
+            schema = SchemaTransformer.transformSchema(schema, typeVisitor);
+        }
+
+        GraphQL.Builder builder = GraphQL.newGraphQL(schema);
+        final List<Instrumentation> instrumentations = this.instrumentations.build();
+        if (!instrumentations.isEmpty()) {
+            builder = builder.instrumentation(new ChainedInstrumentation(instrumentations));
+        }
+
+        final List<GraphQLConfigurator> graphQLBuilders = graphQLBuilderConsumers.build();
+        for (GraphQLConfigurator configurer : graphQLBuilders) {
+            configurer.configure(builder);
+        }
+
+        final DataLoaderRegistry dataLoaderRegistry = new DataLoaderRegistry();
+        final List<Consumer<? super DataLoaderRegistry>> dataLoaderRegistries =
+                dataLoaderRegistryConsumers.build();
+        for (Consumer<? super DataLoaderRegistry> configurer : dataLoaderRegistries) {
+            configurer.accept(dataLoaderRegistry);
+        }
+        return new DefaultGraphQLService(builder.build(), dataLoaderRegistry, useBlockingTaskExecutor);
+    }
+
+    private static TypeDefinitionRegistry typeDefinitionRegistry(List<File> schemaFiles) {
+        final TypeDefinitionRegistry registry = new TypeDefinitionRegistry();
+        final SchemaParser parser = new SchemaParser();
+        if (schemaFiles.isEmpty()) {
+            schemaFiles = defaultSchemaFiles();
+        }
+        if (schemaFiles.isEmpty()) {
+            throw new IllegalStateException("Not found schema file(s)");
+        }
+
+        logger.info("Found schema files: {}", schemaFiles);
+        schemaFiles.forEach(it -> registry.merge(parser.parse(it)));
```

Comment:
nit: Too Kotlin style. 🤣
```suggestion
        schemaFiles.forEach(file -> registry.merge(parser.parse(file)));
```